client/qiskit_serverless/core/job.py

```diff
@@ -306,66 +323,38 @@ def upload(self, program: QiskitFunction):
                 f"Entrypoint file [{program.entrypoint}] does not exist "
                 f"in [{program.working_dir}] working directory."
             )
-        self._patterns[program.title] = {
-            "title": program.title,
-            "entrypoint": program.entrypoint,
-            "working_dir": program.working_dir,
-            "env_vars": program.env_vars,
-            "arguments": json.dumps({}),
-            "dependencies": json.dumps(program.dependencies or []),
-        }
+        self._patterns.append(
+            {
+                "title": program.title,
+                "entrypoint": program.entrypoint,
+                "working_dir": program.working_dir,
+                "env_vars": program.env_vars,
+                "arguments": json.dumps({}),
+                "dependencies": json.dumps(program.dependencies or []),
+            }
+        )
         return program.title
 
-    def run_existing(  # pylint: disable=too-many-locals
+    def run_existing(
         self,
         program: Union[str, QiskitFunction],
         arguments: Optional[Dict[str, Any]] = None,
         config: Optional[Configuration] = None,
     ):
-        if isinstance(program, QiskitFunction):
-            title = program.title
-        else:
-            title = str(program)
-
-        saved_program = self._patterns[title]
-        if saved_program["dependencies"]:
-            dept = json.loads(saved_program["dependencies"])
-            for dependency in dept:
-                subprocess.check_call(
-                    [sys.executable, "-m", "pip", "install", dependency]
-                )
-        arguments = arguments or {}
-        env_vars = {
-            **(saved_program["env_vars"] or {}),
-            **{OT_PROGRAM_NAME: saved_program["title"]},
-            **{"PATH": os.environ["PATH"]},
-            **{ENV_JOB_ARGUMENTS: json.dumps(arguments, cls=QiskitObjectsEncoder)},
-        }
-
-        with Popen(
-            ["python", saved_program["working_dir"] + saved_program["entrypoint"]],
-            stdout=subprocess.PIPE,
-            stderr=subprocess.PIPE,
-            universal_newlines=True,
-            env=env_vars,
-        ) as pipe:
-            status = "SUCCEEDED"
-            if pipe.wait():
-                status = "FAILED"
-            output, _ = pipe.communicate()
-        results = re.search("\nSaved Result:(.+?):End Saved Result\n", output)
-        result = ""
-        if results:
-            result = results.group(1)
-
-        job = Job(job_id=str(uuid4()), job_client=self)
-        entry = {"status": status, "logs": output, "result": result, "job": job}
-        self._jobs[job.job_id] = entry
-        return job
+        warnings.warn(
```

Comment:
This method should not have been used by the user directly.  Can we just take out?